packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -425,6 +426,9 @@ class CupertinoTextField extends StatefulWidget {
   /// {@macro flutter.widgets.scrollable.dragStartBehavior}
   final DragStartBehavior dragStartBehavior;
 
+  /// {@macro flutter.material.textField.scrollPhysics}
```

Comment:
This macro isn't referring to anything defined. 

I would also not cross reference things in cupertino to material docs. Since future maintainers of the material doc might add more material related comments and not realize it's also referenced in cupertino. If possible, move the doc to widgets and reference in both material and cupertino. Otherwise, just copy the doc here.

Reply:
Thanks, good catch.  I set up the template it EditableText and referenced that in material and cupertino text fields and that seems a lot better.